### PR TITLE
Proxy/Bundle UX fixes

### PR DESCRIPTION
**Pull request by Samwalton9**
A handful of fixes based on the UI/UX PRs that just went up on staging:
 - Reduced logo size on the homepage
 - Did wikicode->html properly for partner descriptions on homepage
 - Fixed adding proxy URL for Bundle partners/streams
 - Reworded the successful-application message to point to a more appropriate place
 - Hid the Extend button in My Library for Bundle authorizations
 - Refactored Authorization functions to remove unused `is_renewable`, adding `is_bundle`

### CODE REVIEW

**Comment by Samwalton9** [comment]
This is good for review & merge at any time; if subsequent feedback comes in I'll just open fresh PRs if this is merged.

**Comment by Samwalton9** [comment]
Note to self: Check cache with a different language

File: TWLight/templates/home.html [hunk 1]
```diff
@@ -23,7 +25,7 @@
         </a>
       {% else %}
         <a href="{% url 'oauth_login' %}" class="btn btn-primary btn-block">
-          <span class="glyphicon glyphicon-off"></span>&nbsp;
+          <span class="glyphicon glyphicon-off"></span>&nbsp
```
Comment (jsnshrmn) [inline TWLight/templates/home.html]: why this change? `&nbsp;` is a valid html entity, but `&nbsp` isn't. Was it rendering incorrectly?

**Comment by jsnshrmn** [comment]
> 
> 
> Note to self: Check cache with a different language

You are handling language correctly, based on a readthrough.

File: TWLight/templates/home.html (hunk 1 above)
Comment (jsnshrmn) [reply TWLight/templates/home.html]: Once this is resolved (whether a change is required or not), feel free to merge.

Comment (Samwalton9) [reply TWLight/templates/home.html]: Oh, weird. PyCharm implied this was an end-of-line space, I think. I'll just fix this on Github and merge now.
